app/assets/javascripts/application.js

```diff
@@ -8,5 +8,5 @@ require(['jquery', 'componentLoader', 'eventsWithPromises'], function ($, compon
 require(['jquery'], function ($) {
   require(['AdviserAjaxCall']);
   require(['DataTransform']);
-  require(['FilterTable']);
+  require(['MultiTableFilter']);
```

Comment:
Really we should be using the componentLoader for this stuff, but appreciate that the other Javascript isn't written to Dough's interface so not to worry. Perhaps one for the tech debt list, but not high priority.